asteroid/foo.py

```diff
@@ -0,0 +1,162 @@
+import os
```

Comment:
I didn't know where to stick all of this, so it lives in `asteroid.foo` for now :-D

Reply:
Also not clear to me where to put this yet ^^